src/main/java/org/spongepowered/api/world/extent/Extent.java

```diff
@@ -131,6 +132,28 @@ default Vector3i getHighestPositionAt(Vector3i position) {
     }
 
     /**
+     * Returns the y level that precipitation ends falling in the given column.
+     *
+     * <p>A value is still returned for columns in biomes which do not
+     * receive precipitation.</p>
+     *
+     * @return the y level that precipitation ends
```

Comment:
In both methods, wouldn't hurt to change the "the" to have a capital T. Seems to fit better with usual style across Sponge.